src/sentry/integrations/bitbucket/client.py

```diff
@@ -0,0 +1,198 @@
+from __future__ import absolute_import
+
+import datetime
+import jwt
+
+from unidiff import PatchSet
+
+from six.moves.urllib.parse import urlparse
+
+from sentry.utils.http import absolute_uri
+from sentry.integrations.atlassian_connect import get_query_hash
+from sentry.integrations.client import ApiClient
+
+BITBUCKET_KEY = '%s.bitbucket' % urlparse(absolute_uri()).hostname
+
+
+class BitbucketAPIPath(object):
+    """
+    All UUID's must be surrounded by culybraces.
```

Comment:
what exactly do you mean by this?

Reply:
Oh so actually bitbucket (2.0 anyway) will accept UUID's as well. I don't know if outside developers use it and/or if I user really wanted to they could use uuid's as well. https://developer.atlassian.com/bitbucket/api/2/reference/resource/repositories/%7Busername%7D/%7Brepo_slug%7D 

I can remove that comment if it doesn't make sense for us though!

Reply:
It's important to note that while a username potentially could change in the future, a `uuid` will not. That's the advantage bitbucket sites of using `uuid`'s.